CMS-Enterprise/easi-app
# [EASI-4468] related systems column

# EASI-4468

## Description

- add `Related systems` column

<img width="1194" alt="Screenshot 2024-08-26 at 12 38 23" src="https://github.com/user-attachments/assets/8d3ae608-3c3f-445b-9a5c-267df7af8d0f">

## How to test this change

- this new column only appears on the home page under `My open requests`
- create a system intake and attribute it to one or more systems
- create a TRB requests and attribute it to one or more systems
- nav to home page and cofirm the related systems are listed out and comma separated

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
